erpnext/patches/v13_0/set_per_picked_in_sales_order.py

```diff
@@ -0,0 +1,24 @@
+import frappe
+
+def execute():
+	frappe.reload_doc('selling', 'doctype', 'sales_order', force=True)
+	frappe.reload_doc('stock', 'doctype', 'pick_list_item', force=True)
+	sales_orders = frappe.get_all("Sales Order",
+		filters={"docstatus": 1},
+		fields=["per_delivered", "per_billed"])
+
+	for order in sales_orders:
+		pick_list_items = frappe.get_all("Pick List Item",
+			filters={"docstatus": 1, "sales_order": order.name},
```

Comment:
You're not getting the `name` field in the `get_all`, so none of the code below this will work.